line/line.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "../point/point.h"
+
+namespace miit::figure
+{
+	class line
+	{
+	public:
+		line(const point& start, const point& end);
+
+		const point& getStart() const;
+		const point& getEnd() const;
+	private:
+		const point& start;
+		const point& end;
```

Comment:
Храните указателем или в Вашем случае можно копированием (объект "легкий").
[Avoid reference members, because they restrict what the implementation of a class can do (including, as you mention, preventing the implementation of an assignment operator) and provide no benefits to what the class can provide.](https://stackoverflow.com/questions/892133/should-i-prefer-pointers-or-references-in-member-data)